Contest_API.md

```diff
@@ -1480,6 +1480,43 @@ Returned data:
 ]
 ```
 
+### Commentary
+
+Commentary on events happening in the contest
+
+The following endpoints are associated with commentary:
+
+| Endpoint                         | Mime-type        | Required? | Source @WF | Description                                                                    |
+| -------------------------------- | ---------------- | --------- | ---------- | ------------------------------------------------------------------------------ |
+| `/contests/<id>/commentary`      | application/json | no        | not used   | JSON array of all commentary with elements as defined in the table below       |
+| `/contests/<id>/commentary/<id>` | application/json | no        | not used   | JSON object of a single commentary with elements as defined in the table below |
+
+JSON elements of award objects:
+
+| Name          | Type        | Required? | Nullable? | Source @WF | Description |
+| ------------- | ----------- | --------- | --------- | ---------- | ----------- |
+| id            | ID          | yes       | no        | not used   | identifier of the commentary. |
+| time          | TIME        | yes       | no        | not used   | time of the commentary message. |
+| contest\_time | RELTIME     | yes       | no        | not used   | contest time of the commentary message. |  
+| message       | string      | yes       | no        | not used   | commentary message text. May contain special tags for [teams](#teams) and [problems](#problems) on the format `#t<team ID>` and `#p<problem ID>` respectively.|
```

Comment:
Considering that message is free text, having an escape rule for the literal # is probable a good idea. Perhaps `##` or `\#`.